hbase_sql/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright (c) 2012 - 2020 Splice Machine, Inc.
+  ~ Copyright (c) 2012 - 2019 Splice Machine, Inc.
```

Comment:
update year

Reply:
Fixed in new commit [82fba10](https://github.com/splicemachine/spliceengine/pull/3391/commits/82fba10d7b75dfc54b03624186a18f9b18e50c9d)